<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="WorldGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="WorldTerminalView" name="terminalView" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="enlargeBtn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/img/plus.png</normaloff>:/img/plus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shrinkBtn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/img/minus.png</normaloff>:/img/minus.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="strategyBtn">
        <property name="text">
         <string>Run strategy</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="switchViewBtn">
        <property name="text">
         <string>Switch view</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Set animation speed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="speedSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWorld">
    <property name="title">
     <string>World</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLoad_world"/>
   </widget>
   <addaction name="menuWorld"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_world">
   <property name="text">
    <string>Load world...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WorldTerminalView</class>
   <extends>QWidget</extends>
   <header>terminalview/worldterminalview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WorldGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview/worldgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
